src/DI/ImageStorageExtension.php

```diff
@@ -10,40 +10,49 @@ class ImageStorageExtension extends CompilerExtension
 
 	/** @var mixed[] */
 	private $defaults = [
-		'data_path'          => '%wwwDir%/../public/data',
-		'data_dir'           => 'data',
-		'algorithm_file'     => 'sha1_file',
-		'algorithm_content'  => 'sha1',
-		'quality'            => 85,
-		'default_transform'  => 'fit',
+		'data_path' => '%wwwDir%/../public/data',
+		'data_dir' => 'data',
+		'algorithm_file' => 'sha1_file',
+		'algorithm_content' => 'sha1',
+		'quality' => 85,
+		'default_transform' => 'fit',
 		'noimage_identifier' => 'noimage/03/no-image.png',
-		'friendly_url'       => false,
+		'friendly_url' => false,
```

Comment:
You can remove whole defaults, they are not used anymore and are parts of schema (`'friendly_url' => Nette\Schema\Expect::bool(false)`) etc.

Reply:
Yea i see it now, sorry i did't look at Expect class definition :(

Reply:
It's not easy to migrate, schema works absolutelly different way. Feel free to contact me at [slack channel](https://pehapkari.herokuapp.com) of pehapkari.cz for further help